library/network_connections.py

```diff
@@ -2084,6 +2089,25 @@ def run_action_present(self, idx):
                 % (con_cur.get_id(), con_cur.get_uuid()),
             )
 
+        if (
```

Comment:
@thom311 please take a look it check if this is the right way to figure out if `ca_path` is working with NetworkManager.